src/tracker.js

```diff
@@ -63,29 +63,30 @@ class Tracker {
           timeLeft = timeLeft - Math.round((currentTime - Date.now()) / 1000)
 
           if(timeLeft > 0) {
-            timeout = setTimeout(() => this.promise.reject(TIMEOUT), timeLeft)
+            // timeout = setTimeout(() => this.promise.reject(TIMEOUT), timeLeft)
```

Comment:
I believe you forgot to uncomment this line. I think we also need tests for this feature.